frontend/src/components/RequestDialog.vue

```diff
@@ -133,32 +135,38 @@ const submitRequestTransaction = async (formResult: {
   if (!ethereumProvider.value.signer.value) {
     throw new Error('No signer available!');
   }
-  const request: Request = {
-    targetChainId: Number(formResult.toChainId.value),
-    sourceTokenAddress: formResult.tokenAddress.value,
-    sourceChainId: Number(formResult.fromChainId.value),
-    targetTokenAddress: getTargetTokenAddress(
-      formResult.toChainId.value,
-      formResult.tokenAddress.label,
-    ),
-    targetAddress: formResult.toAddress,
-    amount: formResult.amount,
-  };
-
-  requestMetadata.value = {
-    state: requestState,
-    tokenSymbol: formResult.tokenAddress.label,
-    sourceChainName: formResult.fromChainId.label,
-    targetChainName: formResult.toChainId.label,
-    targetAddress: request.targetAddress,
-    amount: formResult.amount,
-    fee: feesEther.value,
-  };
-  request.fee = fee.value;
-
-  await executeRequestTransaction(request, ethereumProvider.value.signer.value);
-
-  await executeWaitFulfilled(request, requestState);
+
+  try {
```

Comment:
Nitpick: Why did you put so much into the `try` block? Wouldn't it be sufficient to put the `executeRequestTransaction` and `executeWaitFulfilled` in it? In theory the other code should never fail. Buf if it does, I don't think we want to catch it and declare as simple transaction error without a stack trace etc.

Reply:
Nothing specific! I have just surrounded the code inside the function with `try-catch` because I used to use it on the whole body and not for the certain parts that may fail. Not a good or a bad habit :D, but wrapped the async calls only as this may make the code more readable!

Reply:
I don't get the last part. But I assume you made the best decision.